src/main/groovy/edu/oregonstate/mist/directoryapi/DirectoryEntityDAO.groovy

```diff
@@ -71,24 +71,12 @@ class DirectoryEntityDAO {
      *
      * @param ldapConfiguration
      */
-    public DirectoryEntityDAO(Map<String,Object> ldapConfiguration) {
+    public DirectoryEntityDAO(Map<String,Object> ldapConfiguration,
+                              PooledConnectionFactory ldapConnectionPool) {
+
         LDAP_URL = (String)ldapConfiguration.get('url')
```

Comment:
I don't think we need the url in this class anymore. Also, could you remove some of the unused imports in this file?

Reply:
Fixed.